src/components/reusable/modal/modal.stories.js

```diff
@@ -32,6 +32,8 @@ const args = {
   destructive: false,
   okDisabled: false,
   hideFooter: false,
+  hideSecondaryButton: false,
+  hideCancelButton: false,
```

Comment:
IMO default will be **Ok** and **Cancel**. **Secondary** is optional & set this prop. to `showSecondaryButton = false;` as default;

Also if you wanna keep **Secondary** button then there could be a separate story named with secondary button to easily identify we have three button variant.  

<img width="528" alt="Screenshot 2024-04-26 at 3 50 43 PM" src="https://github.com/kyndryl-design-system/shidoka-applications/assets/140634142/6f12ebc0-a532-414b-abd6-d070ee66dcc3">

Reply:
Regarding the new designs, Cancel button also can be optional. But it is there by default most of the time. So having hideCancelButton and showSecondaryButton is confusing.

Reply:
Think if user not used `hideSecondaryButton` prop then by default the Secondary button appears at their end.

Reply:
I think I agree the secondary button should be hidden by default.